src/container/srv_target.c

```diff
@@ -669,11 +685,18 @@ cont_close_one_rec(struct cont_tgt_close_rec *rec)
 		return 0;
 	}
 
-	D_DEBUG(DF_DSMS, DF_CONT": closing: hdl="DF_UUID" hce="DF_U64"\n",
+	D_ASSERT(hdl->sch_cont != NULL);
+
+	if (hdl->sch_cont->sc_closing)
+		resent = true;
+
+	D_DEBUG(DF_DSMS, DF_CONT": closing (%s): hdl="DF_UUID" hce="DF_U64"\n",
 		DP_CONT(hdl->sch_pool->spc_uuid, hdl->sch_cont->sc_uuid),
-		DP_UUID(rec->tcr_hdl), rec->tcr_hce);
+		resent ? "resent" : "new", DP_UUID(rec->tcr_hdl), rec->tcr_hce);
```

Comment:
Lustre uses "resend" from the beginning...

Reply:
Do we have to continue the atrocity? :-)

Reply:
Sorry because of my bad English expressing. My original idea was to use an adjective, but I am not sure which one is suitable for that. On the other hand, I used to Lustre naming style, "resend" is just one of them. As my understanding, sometimes, the past participle of a verb can used as adjective, right? That is why I use "resent" here. But I am fine to rename it to avoid confusing.